app/test_main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "url, connection, expected",
+    [
+        pytest.param(
+            True, True, "Accessible",
+            id="Access granted"
+        ),
+        pytest.param(
+            False, False, "Not accessible",
+            id="Access denied"
+        ),
+        pytest.param(
+            False, True, "Not accessible",
+            id="Access denied"
+        ),
+        pytest.param(
+            True, False, "Not accessible",
+            id="Access denied"
+        )
+
+    ]
+)
+def test_can_access_google_page(
+        url: str,
```

Comment:
```suggestion
        url: bool,
```
Wrong type annotations